wiki/Ranking_Criteria/osu!/en.md

```diff
@@ -17,18 +19,17 @@ General
 -   **Combo colors must not blend in with the map's background/storyboard/video.** This is so the approach circles are always visible to the player.
 -   **There must be a new combo either on or after a group of spinners**, so that the objects on opposing sides of the spinner(s) don't form a single combo. (osu! checks this automatically unless you are editing .osu files by hand.)
 
-Skinning
---------
+### Skinning
 
 -   **Spinner circles must have a clearly visible center** to make sure players have something to effectively spin around while playing through the map.
 -   **Spinner circles must be clearly visible** (i.e. they should not be excessively transparent or near-invisible). This will ensure that players have a visible aid to help them spin consistently.
 
 Guidelines
-==========
+-----------
 
 Guidelines are important and should be followed in most maps. However, they are NOT rules, so they may be broken in **special cases**. If you want to break a guideline, ask yourself this: Does what I'm about to do make sense? Is it more fun to play like this compared to sticking to the guidelines? If you answer yes to both these questions, then it is probably okay.
 
--   **Beat placement should follow the time-distance equality rule** (using [distance snap](Distance_Snap "wikilink") while placing beats enforces this). This means that you shouldn't have some fast notes with a huge distance between them followed by closely-placed (but slow) notes. This gets really confusing if the player isn't the person who designed the beatmap! Having some logic to the beat placement should be one of your goals.
+-   **Beat placement should follow the time-distance equality rule** (using [distance snap](/wiki/Beatmap_Editor/Distance_Snap) while placing beats enforces this). This means that you shouldn't have some fast notes with a huge distance between them followed by closely-placed (but slow) notes. This gets really confusing if the player isn't the person who designed the beatmap! Having some logic to the beat placement should be one of your goals.
 -   **Avoid placing hitobjects under the HP/life bar** at the top of the screen, as this can greatly impair the visibility of notes and approach circles to the player. It doesn't hurt to use the default skin to check this!
 -   **Try to use the same tickrate on every difficulty** as it is a property of the music rather than the mapping. Using high tickrates to increase score/combo/difficulty is senseless.
 -   **Use new combos often and intelligently.** Using a new combo every few beats wouldn't make sense, but you also want to avoid long combos that continue for many measures.
```

Comment:
line 37:
- `[Post Spinner Note](/wiki/Mapping_Techniques/Spinners/#post-spinner-note)`
- `[Pre Spinner Note](/wiki/Mapping_Techniques/Spinners/#pre-spinner-note)`

add a line to the end of the file